_drafts/conda.md

```diff
@@ -0,0 +1,698 @@
+and distribute a new VM image each time a dependency changes, or hope that
+everyone knows when and how to update the system (and does it).
+
+## Example Conda Walk-through
+
+In this part of the post, we will set up Conda on our local machine and set up
+an environment that is able to build, flash, and debug an nRF52 device.
+
+If you'd like to follow along, you may download the Interrupt repo and navigate
+to the example directory.
+
+```
+$ git clone git@github.com:memfault/interrupt.git
+$ cd interrupt/example/using-conda/
+```
+
+> Before we begin, I want to bring comfort to anyone skeptical about installing
+> yet another tool. Conda is the opposite of invasive, and everything is
+> installed into a single directory, `$HOME/miniconda3` by default. If you want
+> to uninstall Conda, deleting that directory will be all that is necessary.
+
+### Installing Conda
+
+To install the `conda` client on a machine, the recommended way is to download
+the interactive
+[Miniconda installer](https://docs.conda.io/en/latest/miniconda.html) and run it
+through `bash`. The following steps apply to both Linux and macOS. If you are
+using Windows, there is a Miniconda installer that can be downloaded from the
+same page.
+
+```
+$ wget https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
+$ bash Miniconda3-latest-MacOSX-x86_64.sh
+```
+
+The final step in the installer is to make sure the
+`$HOME/miniconda3/etc/profile.d/conda.sh` script is loaded in the shell. When
+the following prompt is given:
+
+```
+Do you wish the installer to initialize Miniconda3
+by running conda init? [yes|no]
+```
+
+I suggest saying `no` and modifying the appropriate shell startup file
+(`.bashrc` or `.zshrc`, etc.) to contain the following.
+
+```
+. "$HOME/miniconda3/etc/profile.d/conda.sh"
+```
+
+If everything was successful, after reloading our shell, we can now run the
+`conda` command in the shell.
+
+```
+$ conda --version
+conda 4.7.12
+```
+
+### Creating the Environment
+
+The next step is to create an isolated environment in which we can install
+packages into. We will create an environment called `example`.
+
+```
+$ conda create -n example
+Collecting package metadata (current_repodata.json): done
+Solving environment: done
+
+## Package Plan ##
+
+  environment location: /Users/tyler/miniconda3/envs/example
+
+Proceed ([y]/n)? y
+
+Preparing transaction: done
+Verifying transaction: done
+Executing transaction: done
+#
+# To activate this environment, use
+#
+#     $ conda activate example
+#
+# To deactivate an active environment, use
+#
+#     $ conda deactivate
+
+```
+
+Now, we are able to load up our environment using:
+
+```
+$ conda activate example
+```
+
+Our current environment has zero packages installed. We can double check this by
+running `conda list`.
+
+```
+(example)
+$ conda list
+# packages in environment at /Users/tyler/miniconda3/envs/example:
+#
+# Name                    Version                   Build  Channel
+
+```
+
```

Comment:
Maybe add a link to the full command reference for `create`? https://docs.conda.io/projects/conda/en/latest/commands/create.html